src/components/CustomAppDetails/Versions.js

```diff
@@ -0,0 +1,271 @@
+                    </TableCellHead>
+                    <TableCellHead>{i18n.t('Upload date')}</TableCellHead>
+                    <TableCellHead></TableCellHead>
+                </TableRowHead>
+            </TableHead>
+            <TableBody>
+                {versions.map(version => (
+                    <TableRow key={version.id}>
+                        <TableCell>{version.version}</TableCell>
+                        <TableCell>{version.channel}</TableCell>
+                        <TableCell>
+                            {renderVersionRange(
+                                version.minDhisVersion,
+                                version.maxDhisVersion
+                            )}
+                        </TableCell>
+                        <TableCell>
+                            {moment(version.created).format('L')}
+                        </TableCell>
+                        <TableCell>
+                            <Button
+                                small
+                                secondary
+                                className={styles.installBtn}
+                                disabled={version.version === installedVersion}
+                                onClick={() => onVersionInstall(version)}
+                            >
+                                {version.version === installedVersion
+                                    ? i18n.t('Installed')
+                                    : i18n.t('Install')}
+                            </Button>
+                            <a download href={version.downloadUrl}>
+                                <Button small secondary>
+                                    {i18n.t('Download')}
+                                </Button>
+                            </a>
+                        </TableCell>
+                    </TableRow>
+                ))}
+            </TableBody>
+        </Table>
+    )
+}
+
+VersionsTable.propTypes = {
+    versions: PropTypes.array.isRequired,
+    onVersionInstall: PropTypes.func.isRequired,
+    installedVersion: PropTypes.string,
+}
+
+const Versions = ({ installedVersion, versions, reloadPage }) => {
+    const [channelsFilter, setChannelsFilter] = useState(new Set(['Stable']))
+    const [dhisVersionFilter, setDhisVersionFilter] = useState('')
+    const installSuccessAlert = useAlert(i18n.t('App installed successfully'), {
+        success: true,
+    })
+    const installErrorAlert = useAlert(
+        ({ error }) =>
+            i18n.t('Failed to install app: {{errorMessage}}', {
+                errorMessage: error.message,
+                nsSeparator: null,
+            }),
+        { critical: true }
+    )
+    const { apiVersion } = useConfig()
+    const { installVersion } = useApi()
+    const latestDhisVersion = `2.${apiVersion}`
+    const dhisVersions = [
+        latestDhisVersion,
+        `2.${apiVersion - 1}`,
+        `2.${apiVersion - 2}`,
+    ]
```

Comment:
This has a few different issues:

1. As mentioned above, having a select doesn't really make sense at all when the point is to install into the current instance so this should probably be removed
2. This actually gets the current server version which **is almost never actually the latest version**, so showing the current version and two previous releases is incorrect.
3. When referencing the version of the current server we should use [`serverVersion`](https://github.com/dhis2/app-runtime/blob/master/services/config/src/types.ts#L1-L6) from the `config` object instead of `apiVersion` (they currently match but this might not always be true)